src/fairchem/core/common/utils.py

```diff
@@ -1231,11 +1231,23 @@ def scatter_det(*args, **kwargs):
     return out
 
 
-def get_commit_hash():
+def get_commit_hash() -> str:
+    core_hash = get_commit_hash_for_repo(fairchem.core.__path__[0])
+    experimental_hash = None
+    try:
+        experimental_hash = get_commit_hash_for_repo(fairchem.experimental.__path__[0])
+        return f"core:{core_hash},experimental:{experimental_hash}"
+    except (NameError, AttributeError):
+        return f"core:{core_hash},experimental:NA"
```

Comment:
Just curious, why return as a single string? to log it as a single item?

Reply:
I think leaving it as a unique string that can be decoded by a human to the correct commits used is what we want in the end of the day. I didn't want to pass that off to a separate function. An earlier commit did that, but its more complicated than it needs to be. If we use it more places, maybe its worth while making sub functions? I'm not super sold either way. Either works, as long as we log both commits :)